reference.html

```diff
@@ -118,184 +124,184 @@ <h2 id="errors-and-exceptions"><a href="07-errors.html">Errors and Exceptions</a
 <h2 id="glossary">Glossary</h2>
 <dl>
 <dt><span id="additive-color-model">additive color model</span></dt>
-<dd>A way to represent colors as the sum of contributions from primary colors such as <a href="#rgb">red, green, and blue</a>.
+<dd><p>A way to represent colors as the sum of contributions from primary colors such as <a href="#rgb">red, green, and blue</a>.</p>
```

Comment:
> Are those due to use of a different pandoc version for the translation?

Probably yes.

> I have pandocfilters version 1.2.3 installed.

This is a very old version of Pandoc. The last version is 1.13.2. You can download the last version at https://github.com/jgm/pandoc/releases.